pkg/util/hostname.go

```diff
@@ -292,3 +294,17 @@ func isHostnameCanonicalForIntake(hostname string) bool {
 	}
 	return true
 }
+
+// Gets a valid EC2 hostname
```

Comment:
let's follow the convention:

```suggestion
// getValidEC2Hostname gets a valid EC2 hostname
```